includes/class-ssp-frontend.php

```diff
@@ -171,7 +171,14 @@ public function content_meta_data( $content = '' ) {
 
 		$podcast_post_types = ssp_post_types( true );
 
-		if ( in_array( $post->post_type, $podcast_post_types ) && ! is_feed() && ! isset( $_GET['feed'] ) ) {
+		$player_visibility = get_option( 'ss_podcasting_player_content_visibility' );
```

Comment:
This `get_option` call needs a default value of `all` just to make sure that it catches every scenario ([function docs here](https://developer.wordpress.org/reference/functions/get_option/)). Other than that, this code is all good with me.